.travis.yml

```diff
@@ -17,7 +17,7 @@ cache:
 env:
   matrix:
     # django
-    - DJANGO_VERSION=1.8 DJANGO_SETTINGS_MODULE="contributr.settings"
+    - DJANGO_VERSION=1.8 DJANGO_SETTINGS_MODULE="contributr.settings.testing"
```

Comment:
line 20 says that the settings are in a file called `testing.py` within the folder `contribute/settings`. The `settings` folder should have a `__init__.py` file for it to be used as a module.